peep.py

```diff
@@ -362,9 +362,11 @@ def package_finder(argv):
     # Carry over PackageFinder kwargs that have [about] the same names as
     # options attr names:
     possible_options = [
-        'find_links', FORMAT_CONTROL_ARG, 'allow_external', 'allow_unverified',
```

Comment:
I don't think we ought to remove allow_external and allow_all_external; there's nothing wrong (from a security standpoint—especially when checking hashes) with downloading from sites other than PyPI. Can we stick these in a branch? We might as well bring back allow_unverified while we're at it.